src/main/java/eu/arrowhead/blacklist/collector/DatabaseCollectorDriver.java

```diff
@@ -0,0 +1,36 @@
+package eu.arrowhead.blacklist.collector;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import eu.arrowhead.common.collector.ICollectorDriver;
+import eu.arrowhead.common.exception.ArrowheadException;
+import eu.arrowhead.common.model.ServiceModel;
+
+public class DatabaseCollectorDriver implements ICollectorDriver {
```

Comment:
I don't see why this is needed. If the BL system has to call other systems' services it should use the HttpCollector.